app/components/QuestionForm/QuestionTypeField.js

```diff
@@ -26,15 +26,15 @@ const styles = singleCommunityStyles();
 
 export const QUESTION_TYPES = {
   GENERAL: {
-    value: '1',
+    value: 1,
```

Comment:
It is better to use `POST_TYPE` constant instead of harcoding values in multiple places.

`POST_TYPE.expertPost, POST_TYPE.generalPost, POST_TYPE.tutorial`